packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -78,6 +91,10 @@ To display placeholder text within the \`Select\` you must provide an option tha
 Any \`Select\` without a default selected option should provide placeholder text. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Use sentence casing and don't include punctuation at the end of the prompt.
```

Comment:
>To display placeholder text within the \`Select\` you must provide an option that has the \`disabled\`, \`selected\` and \`hidden\` attributes set. 

Curious if you think it's worth going into more detail on the need to specify `selected`. Now you could theoretically provide a placeholder that's not selected by default but is still selectable via the clear button. Do we want to clarify that `selected` isn't needed unless you want the placeholder to be visible by default? Or do you think that an initially unselected placeholder is such an uncommon use case that it's not worth describing?

Reply:
I was personally thinking the latter, but I don't think it would hurt to get into a little more detail.